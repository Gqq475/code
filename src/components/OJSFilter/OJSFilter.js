import React from 'react'
import classes from './OJSFilter.scss'

type Props = {

}
export class OJSFilter extends React.Component {
  props: Props;

  render () {
    return (
      <div className={classes['OJSFilter-container']}>
      OJSFilter
      </div>
    )
  }
}

export default OJSFilter
