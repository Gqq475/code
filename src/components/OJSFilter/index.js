import OJSFilter from './OJSFilter'
export default OJSFilter
