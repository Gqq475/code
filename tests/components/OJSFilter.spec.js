import React from 'react'
import OJSFilter from 'components/OJSFilter/OJSFilter'

describe('(Component) OJSFilter', () => {
  it('render normally', () => {
    const wrapper = shallow(
      <OJSFilter />
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
