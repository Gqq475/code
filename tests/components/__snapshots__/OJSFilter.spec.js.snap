// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) OJSFilter render normally 1`] = `
<div
  className="OJSFilter-container"
>
  OJSFilter
</div>
`;
